src/Logger.h

```diff
@@ -0,0 +1,47 @@
+// Copyright The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_LOGGER_H_
+#define MUMBLE_LOGGER_H_
+
+#include <memory>
+
+#include <spdlog/spdlog.h>
+
+class QTextEdit;
+
+namespace mumble {
+namespace log {
+	void init(QTextEdit *textBox = nullptr);
+	void prepareToExit();
+
+	template< typename... Args > static void inline trace(spdlog::format_string_t< Args... > fmt, Args &&... args) {
+		spdlog::trace(fmt, std::forward< Args >(args)...);
+	}
```

Comment:
What's the point of wrapping spdlog's functions in our own function when all we do is passing arguments through?

Reply:
1. We can use our own (shorter) and generic namespace (`log::trace()` instead of `spdlog::trace()` in this case).
2. `fatal()` doesn't exist in `spdlog`, it's our own function that mimics `qFatal()`'s behavior (printing the critical error and exiting).
3. Performance is not affected at all because it's an inline function.

Reply:
I'm not concerned about performance but maintenance. 

The namespace is actually longer as we have `mumble::log::*` now.

For fatal error, I think it can be argued that there should be a general function `fatalError` or something like that, that is not in the `log` namespace. His might make it a bit more clear that this is going to do more than log a message. I think this would improve code readability.